test/data/low_income_thresholds.test.ts

```diff
@@ -0,0 +1,47 @@
+import { test } from 'tap';
+import { LOW_INCOME_THRESHOLDS_BY_AUTHORITY } from '../../src/data/low_income_thresholds';
+import {
+  STATE_INCENTIVES_BY_STATE,
+  StateIncentive,
+} from '../../src/data/state_incentives';
+
+function computeIdToIncentiveMap(incentives: StateIncentive[]) {
+  const output: { [index: string]: StateIncentive } = {};
+  for (const incentive of incentives) {
+    output[incentive.id] = incentive;
+  }
+  return output;
+}
+
+test('low-income thresholds are equivalent in JSON config and incentives', async t => {
+  // All incentives have the threshold indicated in their config.
+  const map = computeIdToIncentiveMap(
+    Object.values(STATE_INCENTIVES_BY_STATE).flat(),
+  );
+  Object.entries(LOW_INCOME_THRESHOLDS_BY_AUTHORITY).forEach(
+    ([, stateThresholds]) => {
+      for (const [identifier, thresholds] of Object.entries(stateThresholds)) {
+        for (const incentiveId of thresholds.incentives) {
+          const incentive = map[incentiveId];
+          t.equal(incentive.low_income, identifier);
```

Comment:
This would catch a case where multiple threshold configs point to the same incentive, right?